recipes/configs/llama3_1/kd_single_device.yaml

```diff
@@ -0,0 +1,110 @@
+# Config for single device knowledge distillation in kd_single_device.py
+# using a teacher and student model
+#
+# This config assumes that you've run the following commands before launching
+# this run:
+#   tune download meta-llama/Meta-Llama-3.1-8B-Instruct --output-dir /tmp/Meta-Llama-3.1-8B-Instruct
+#   tune run lora_finetune_single_device --config llama3_1/8B_lora_single_device
```

Comment:
Can you add a bit more detail here? Tbh when I first looked at it I thought you had accidentally just copy-pasted a `tune run` command from another config 😅 . Maybe just add a couple statements explicitly saying something like "Run this to download the model: {tune download ...}. You will then need to fine-tune the teacher model, you can do this with {tune run...}"